claim_ai/test_response_bundle.py

```diff
@@ -0,0 +1,256 @@
+adjudication_bundle = {
+    'resourceType': 'AdjudicationBundle',
+    'entry': [
+        {
+                "resourceType": "ClaimResponse",
+                "created": "2020-11-15",
+                "id": "C670E61A-36F1-4F70-A5D2-6CE2C20457F6",
+                "item": [
+                    {
+                        "adjudication": [
```

Comment:
If this is an example for testing, please look in the specification https://openimis.atlassian.net/wiki/spaces/OP/pages/1790345259/openIMIS-AI+-+6.+Specification#Build-FHIR-ClaimResponse. The ClaimResponse will have only one adjudication item with the description in the specs.

Reply:
Payload changed to meet the specification